resources/monitoring/charts/prometheus-istio/values.yaml

```diff
@@ -1329,20 +1288,8 @@ serverFiles:
           regex: __meta_kubernetes_pod_label_(.+)
           replacement: $1
           action: labeldrop
-        - source_labels: [__meta_kubernetes_namespace]
```

Comment:
You still need to drop the source label like it is done on the block above, don't you? Have you verified that the source labels are not there?